vllm/worker/habana_worker.py

```diff
@@ -356,15 +364,17 @@ def init_worker_distributed_environment(
             "distributed_init_method must be set if torch.distributed "
             "is not already initialized")
     else:
+        backend = 'hccl' if not is_fake_hpu() else 'gloo'
         torch.distributed.init_process_group(
-            backend="hccl",
+            backend=backend,
             world_size=parallel_config.world_size,
             rank=rank,
             init_method=distributed_init_method,
         )
 
     # A small all_reduce for warmup & checking conformance.
-    dummy_tensor_hpu = torch.ones(1).to('hpu')
+    device = 'hpu' if not is_fake_hpu() else 'cpu'
```

Comment:
I've seen this snippet before. Couldn't we wrap it in a helper function? Like hpu_device_str() and move the check inside?

Reply:
Wrapped in a helper function